cockpit/build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.android.tools.build:gradle:3.1.3'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:2.1'
+        classpath files("../CockpitCore/build/libs/CockpitCore-1.1.1-SNAPSHOT.jar")
```

Comment:
what about extracting core version to some variable and use `$core_version` here?